src/inputs/ObjectSearchCreate.tsx

```diff
@@ -99,13 +119,13 @@ class ObjectSearchCreate extends Component<IInputProps> {
           ))}
         </Antd.Select>
         <Antd.Button
+          icon='plus'
```

Comment:
Order these props (especially have `{...this.buttonProps}` come last)

Reply:
I guess it's fine for now because I `pick()` them.
But normally, I wanted the ones at the top to be overridable and the ones at the bottom not to be.